flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/rules/logical/FlinkPruneEmptyRulesTest.scala

```diff
@@ -46,9 +46,20 @@ class FlinkPruneEmptyRulesTest extends TableTestBase {
           FlinkSubQueryRemoveRule.FILTER,
           CoreRules.FILTER_REDUCE_EXPRESSIONS,
           CoreRules.PROJECT_REDUCE_EXPRESSIONS,
-          PruneEmptyRules.FILTER_INSTANCE,
+          CoreRules.FILTER_SET_OP_TRANSPOSE,
+          CoreRules.FILTER_PROJECT_TRANSPOSE,
+          CoreRules.PROJECT_MERGE,
+          CoreRules.PROJECT_FILTER_VALUES_MERGE,
+          PruneEmptyRules.UNION_INSTANCE,
+          PruneEmptyRules.INTERSECT_INSTANCE,
+          PruneEmptyRules.MINUS_INSTANCE,
           PruneEmptyRules.PROJECT_INSTANCE,
-          PruneEmptyRules.JOIN_RIGHT_INSTANCE
+          PruneEmptyRules.FILTER_INSTANCE,
+          PruneEmptyRules.SORT_INSTANCE,
+          PruneEmptyRules.AGGREGATE_INSTANCE,
+          PruneEmptyRules.JOIN_LEFT_INSTANCE,
+          PruneEmptyRules.JOIN_RIGHT_INSTANCE,
+          PruneEmptyRules.EMPTY_TABLE_INSTANCE
```

Comment:
Please add tests for `EMPTY_TABLE_INSTANCE`

Reply:
remove this rule EMPTY_TABLE_INSTANCE becuase flink will not use it